website/content/v1.9/talos-guides/network/siderolink.md

```diff
@@ -13,9 +13,12 @@ SideroLink is a foundation building block of [Sidero Omni](https://www.siderolab
 SideroLink is configured by providing the SideroLink API server address, either via kernel command line argument `siderolink.api` or as a [config document]({{< relref "../../reference/configuration/siderolink/siderolinkconfig" >}}).
 
 SideroLink API URL: `https://siderolink.api/?jointoken=token&grpc_tunnel=true`.
+
 If URL scheme is `grpc://`, the connection will be established without TLS, otherwise, the connection will be established with TLS.
+
 If specified, join token `token` will be sent to the SideroLink server.
-If `grpc_tunnel` is set to `true`, the Wireguard traffic will be tunneled over the same SideroLink API gRPC connection instead of using plain UDP.
+
+If `grpc_tunnel` is set to `true`, the Wireguard traffic will be tunneled over the same SideroLink API gRPC connection instead of using plain UDP. This supports environments where UDP traffic might be restricted, but adds significant overhead to Siderolink communication -- only enable if you must.
```

Comment:
I will clarify this a bit, in fact `grpc_tunnel` is a hint to the SideroLink server whether to use the tunnel or not.

A decision whether to enable GRPC Tunnel is ultimately the SideroLink server decision, and Talos will follow that.